### PR TITLE
releasenotes for 2.10.1

### PR DESCRIPTION
**Pull request by vvolkl**

### CODE REVIEW

File: cpt-releasenotes.rst [hunk 1]
```diff
@@ -1,3 +1,41 @@
+Release Notes for CernVM-FS 2.10.1
+=================================
+
+CernVM-FS 2.10.1 is a patch release, containing several minor bug fixes and improvements.
+As with previous releases, upgrading clients should be seamless just by
+installing the new package from the repository. As usual, we recommend to updateonly a few worker nodes first and gradually ramp up once the new version proves
```
Comment (jblomer) [inline cpt-releasenotes.rst]:
```suggestion
installing the new package from the repository. As usual, we recommend to update only a few worker nodes first and gradually ramp up once the new version proves
```

File: cpt-releasenotes.rst [hunk 2]
```diff
@@ -1,3 +1,41 @@
...
+to work correctly. Please take special care when upgrading a cvmfs client in NFSmode.
```
Comment (jblomer) [inline cpt-releasenotes.rst]:
```suggestion
to work correctly. Please take special care when upgrading a cvmfs client in NFS mode.
```

File: cpt-releasenotes.rst [hunk 3]
```diff
@@ -1,3 +1,41 @@
...
+
+Bug Fixes and Improvements
+--------------------------
+
+  * [server] cvmfs_server check can be extremely slow with many references to deduplicated files (`#3138 <https://github.com/cvmfs/cvmfs/issues/3138>`_)
+  * [server] Make a parallelization limit on ``cvmfs_server snapshot -a`` (`#3064 <https://github.com/cvmfs/cvmfs/issues/3064>`_) 
+  * [server] Succeeding cvmfs_server check should clear status in .cvmfs_status.json (`#3147 <https://github.com/cvmfs/cvmfs/issues/3147>`_)
+  * [server] Reflog always recreated when running ``cvmfs_server check -r -i FQRN`` (`#3123 <https://github.com/cvmfs/cvmfs/issues/3123>`_)
+  * [server] Unable to recover from publish failure (`#3088 <https://github.com/cvmfs/cvmfs/issues/3088>`_)
+  * [server] snapshot data file has doubled data (`#2991 <https://github.com/cvmfs/cvmfs/issues/2991>`_)
```
Comment (jblomer) [inline cpt-releasenotes.rst]:
```suggestion
  * [server] Snapshot data file has doubled data (`#2991 <https://github.com/cvmfs/cvmfs/issues/2991>`_)
```

File: cpt-releasenotes.rst [hunk 4]
```diff
@@ -1,3 +1,41 @@
...
+  * [gw/server]  Allow aborting a transaction when session token is missing (`#3159 <https://github.com/cvmfs/cvmfs/issues/3159>`_)
+  * [client] attr -g logbuffer can overflow (`#2979 <https://github.com/cvmfs/cvmfs/issues/2979>`_)
+  * [client] Misleading "Transfer fuse connection to new mount...success" message (`#2956 <https://github.com/cvmfs/cvmfs/issues/2956>`_)
+  * [client] Error when creating sqlite-backed NFS maps (`#3150 <https://github.com/cvmfs/cvmfs/issues/3150>`_)
+  * [client] ``cvmfs_config fuser`` runs too long on host with afs (`#3090 <https://github.com/cvmfs/cvmfs/issues/3090>`_)
+  * [container] Container conversion does not recognize OCI manifest lists (`#3164 <https://github.com/cvmfs/cvmfs/issues/3164>`_)
+  * [build system] cvmfs-2.9.3 el8 compilation failure (`#2985 <https://github.com/cvmfs/cvmfs/issues/2985>`_)
```
Comment (jblomer) [inline cpt-releasenotes.rst]: I would remove that from the release notes list. It is a fix for a quite narrow case.

**Review by jblomer — COMMENTED** [review]
LGTM! Some typos.

File: cpt-releasenotes.rst (hunk 4 above)
Comment (vvolkl) [reply cpt-releasenotes.rst]:
```suggestion
```

**Pull request merged**